File: 2185_Counting_Words_With_a_Given_Prefix.js
var prefixCount = function(words, pref) {
    let count = 0
    let n = pref.length
    for(let i = 0; i < words.length; i++)
    {
        let s = words[i].slice(0,n)
        if(s == pref) count++
    }
    return count
};

File: 2185_Counting_Words_With_a_Given_Prefix.cpp
class Solution {
public:
    int prefixCount(vector<string>& words, string pref) {
        int n = pref.length(), ans = 0;
        for(string s : words){
            if(s.substr(0,n) == pref) ans++;
        }
        return ans;
    }
};
